src/Microsoft.NetCore.Analyzers/Core/Security/Helpers/SecurityMemberNames.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.NetCore.Analyzers.Security.Helpers
+{
+    internal static class SecurityMemberNames
+    {
+        public const string CreateSignature = "CreateSignature";
```

Comment:
`nameof(CreateSignature)`

Reply:
System.Security.Cryptography.DSA doesn't exist in .NET Standard 1.3 / .NET Core 1.0.

Reply:
I meant change this declaration to `public const string CreateSignature = nameof(CreateSignature);`

Reply:
Oh...got it.